### PR TITLE
replace getJSON with pure XHR/JSON.parse implementation

### PR DESCRIPTION
**Pull request by thisisaaronland**
I tried to replace the remaining jQuery dependencies but got lost in a twisty maze of Leaflet expectations.

The other issue is that this PR may not address older versions of IE, maybe?

### CODE REVIEW

File: leaflet-iiif.js [hunk 1]
```diff
@@ -165,7 +163,66 @@ L.TileLayer.Iiif = L.TileLayer.extend({
...
+
+                    console.log("Failed to parse JSON, because " + e);
+		    console.log(raw);
+                    return false;
+                }
+
+	  onsuccess(data);
+      };
+
+      var onprocess = function(){
```
Comment (mejackreed) [inline leaflet-iiif.js]: Should this be `onprogress`?

File: leaflet-iiif.js [hunk 2]
```diff
@@ -165,7 +163,66 @@ L.TileLayer.Iiif = L.TileLayer.extend({
...
+                var data = undefined;
+
+                try {
+                    data = JSON.parse(raw);
+                }
+
+                catch (e){
+
+                    console.log("Failed to parse JSON, because " + e);
+		    console.log(raw);
```
Comment (mejackreed) [inline leaflet-iiif.js]: Looks like there is a mix of tabs and four spaces. Mind converting over to two spaces?

**Comment by thisisaaronland** [comment]
`onprogress` - yes

I'll sort the spaces out shortly. The next couple days are a bit nutty.

**Comment by mejackreed** [comment]
Popping this here for reference too. Not sure but we could include this plugin instead? https://github.com/calvinmetcalf/leaflet-ajax

**Comment by thisisaaronland** [comment]
Well, it's your party so you should do whatever feels right :D 

Looking over the code with sloppy eyes it's not clear what it gets you – the `ajax.js` code seems like the same basic XHR scaffolding as the PR – but I haven't looked at the rest very carefully.

I guess the larger issue is whether or not you want external dependencies for the iiif-leaflet plugin. If you do (or don't mind having them) then there are worse things that letting jQuery take care of the details for you.

**Comment by mejackreed** [comment]
Closing this now in favor of #77 .

**Pull request closed without merging**